rllib/algorithms/tests/test_node_failures.py

```diff
@@ -137,7 +139,7 @@ def _train(self, *, config, iters, min_reward, preempt_freq):
                 exp_batch_size = config.minibatch_size
             else:
                 exp_batch_size = config.total_train_batch_size
-            self.assertGreaterEqual(avg_batch, exp_batch_size)
+            check(avg_batch, exp_batch_size, rtol=0.1)
```

Comment:
Not sure why we were asserting GreaterEqual here before.
I could not find any explicit guarantees about it being larger than the expected.

I debugged this for a while and could not see anything being wrong with either version (with/without this PR).

Reply:
Probably only the case when `batch_mode="complete_episodes"` ...